src/components/mypage/api.ts

```diff
@@ -0,0 +1,25 @@
+import { axiosInstance } from '@/api';
+
+import { MyPostAPI } from './types';
+
+const SERVER = process.env.NEXT_PUBLIC_API_URL;
+
+export const getPostFeedLimit = async () => {
+  try {
+    const { data } = await axiosInstance.request<MyPostAPI>({
+      method: 'GET',
+      url: `${SERVER}/members/posts`,
```

Comment:
[chore: axiosInstance에 baseURL 적용](https://github.com/prgrms-web-devcourse/Team-6Jeans-Hype-FE/pull/138/files#top)
이 PR이 Merge 될 예정이기 때문에 아래 처럼 수정하셔도 좋을 것 같습니다!
```suggestion
      url: `/members/posts`,
```

Reply:
반영했습니다!